### PR TITLE
Normalize buttons in about:preferences

### LINKED ISSUE #9543: Add browserButton_panelItem to BrowserButton component
## Test plan

1. Open `about:preferences#payments`
2. Click `Add funds`
3. Make sure the three buttons have 180px
4. Open `about:styles#buttons`
5. Make sure there are 2 grouped panel buttons

#9487

----

**Describe the issue you encountered:** Add browserButton_panelItem to BrowserButton component.

- Extra QA steps:
    1.
    2.
    3.

- Any related issues: #9071 #9487

### PR DESCRIPTION
**Pull request by cezaraugusto**
Addresses #9223
Fixes #9543

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Don't need PR #9357 but will preferably land after that

Auditors: @luixxiul
Test Plan:
Buttons inside prefs should look the same across tabs

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by luixxiul** [comment]
The buttons on `bitcoinDashboard.js` are regressed:

screenshot 2017-06-18 15 30 01

See: https://github.com/brave/browser-laptop/pull/9487/commits/ee832b2c883c534dad3dae5c338e2bcde40cbbb6#diff-799b11467a9bfc86f7dbbf11fcaee159R477
````
panelButton: {
    minWidth: '180px !important'
},
````

**Comment by luixxiul** [comment]
Also @cezaraugusto, would you please create an issue to track the TODOs?

**Comment by cezaraugusto** [comment]
thanks, updated

> would you please create an issue to track the TODOs?
sorry but which to-dos?

**Comment by luixxiul** [comment]
> sorry but which to-dos?

https://github.com/brave/browser-laptop/commit/f214325c4381675f8b2e1b2274814d5d64cb63a7#diff-e3eeb751016b2ce9f8278efce585a461R556,
https://github.com/brave/browser-laptop/commit/f214325c4381675f8b2e1b2274814d5d64cb63a7#diff-e3eeb751016b2ce9f8278efce585a461R619,
and https://github.com/brave/browser-laptop/commit/f214325c4381675f8b2e1b2274814d5d64cb63a7#diff-e3eeb751016b2ce9f8278efce585a461R664.

File: js/about/styles.js [hunk 1]
```diff
@@ -312,6 +312,13 @@ class AboutStyle extends ImmutableComponent {
           &lt;BrowserButton iconOnly icon={'{'}globalStyles.appIcons.private{'}'} size='60px' color='red' />
         </Code></Pre>
 
+        <BrowserButton groupedItem secondaryColor panelItem l10nId='panelItem' onClick={this.onEnableAutoplay} />
+        <BrowserButton groupedItem primaryColor panelItem l10nId='panelItem' onClick={this.onEnableAutoplay} />
```
Comment (luixxiul) [inline js/about/styles.js]: Maybe `this.copyToClipboard` and `this.props.showQRcode` look better, which are actually used only on bitcoinDashboard.js for now.

File: js/about/styles.js [hunk 2]
```diff
@@ -312,6 +312,13 @@ class AboutStyle extends ImmutableComponent {
...
+        <Pre><Code>
+          &lt;BrowserButton groupedItem secondaryColor panelItem l10nId='panelItem' onClick={'{this.onEnableAutoplay}'} />{'\n'}
+          &lt;BrowserButton groupedItem primaryColor panelItem l10nId='panelItem' onClick={'{this.onEnableAutoplay}'} />
```
Comment (luixxiul) [inline js/about/styles.js]: Ditto.

**Comment by cezaraugusto** [comment]
ok updated example. re https://github.com/brave/browser-laptop/pull/9487#issuecomment-309323240 I preferred to create a tracking issue for prefs refactoring: #9558, given we'll naturally remove those comments once refactored. lmk if that sounds good to you

**Pull request merged**